src/express-emulator/express.js

```diff
@@ -63,7 +63,7 @@ const app = (function () {
 			body = '';
 		}
 		if (!handlers[method]) {
-			return Bluebird.reject(404);
+			return Bluebird.reject([404, null, null]);
```

Comment:
This now matches up with the rejection value on all the other branches of this promise